R/system.R

```diff
@@ -1262,7 +1262,9 @@ getGoogleBigQueryTables <- function(project, dataset, tokenFileId=""){
     # If we pass large value to max_results (via page_size argument) like 1,000,000, Google BigQuery gives
     # Error: Invalid value at 'max_results.value' (TYPE_UINT32), "1e+06" [badRequest]
     # so set 10,000 as the default value.
-    tables <- bigrquery::list_tables(project, dataset, page_size=10000);
+    dataset <- bigrquery::bq_dataset(project = project, dataset = dataset)
```

Comment:
Does this line actually fetch the data from the table? or just get a reference to the table. I'm worried about downloading millions of records here.